elmslie-core/src/commonMain/kotlin/money/vivid/elmslie/core/utils/DispatcherProvider.kt

```diff
@@ -0,0 +1,5 @@
+package money.vivid.elmslie.core.utils
+
+import kotlinx.coroutines.CoroutineDispatcher
+
+internal expect val DefaultIODispatcher: CoroutineDispatcher
```

Comment:
Better to avoid using Default word here, because we have Default Dispatcher and Default here has another meaning.

Reply:
plus 1